### ISSUE TITLE
ABV Graphics Updating Incorrectly

### ISSUE THREAD

**Issue by thadius856** [RaspberryPints/RaspberryPints#16]
During the CSV->SQL conversion/CSS cleanup, some image updates broke.

In particular, the ABV graphic consistently under-fills. A 5% beer should show one full shot glass, a 10% beer should show two shot glasses, anything higher should show two full shot glasses plus a spilled third. See screeshot:

![untitled](https://f.cloud.github.com/assets/6320180/1866562/fed856da-7835-11e3-8015-814b8511e70e.png)

**Comment by Skemp45** [comment]
ok so first issue is you cant set the abv to anything higher than 9.9 with the way the database is setup, 

length/values set to 2, 1 seems to be the issue, I removed that and then added a beer at 10.5 and it took and it showed the offthechart cup so this is a good start.

**Comment by Skemp45** [comment]
You must change the 2, 1 to 3,1 and this will fix that FYI dont leave it empty

**Comment by Skemp45** [comment]
does anyone have the original file where this worked with the csv?

**Comment by thadius856** [comment]
Yep. Here ya go.

```
                        <?php
                            $numCups = 0;
                            $remaining = round(($abv['Beer'.$i]*20*2), -1, PHP_ROUND_HALF_UP)/2;
                            do{                             
                                if( $remaining < 100 ){
                                    $level = $remaining;
                                }else{
                                    $level = 100;
                                }
                                ?><div class="abv-indicator"><div class="abv-full" style="height:<?php echo $level; ?>%"></div></div><?php                              

                                $remaining = $remaining - $level;
                                $numCups++;
                            }while($remaining > 0 && $numCups < 2);

                            if( $remaining > 0 ){
                                ?><img class="abv-max" src="/img/abv-new/offthechart.png" /><?php
                            }
                        ?>
```

One shot of 80-proof liquor is exactly equal to one 12-oz glass of 5% beer. Exact same alcohol content, that is.

If one shot glass is "100" and two shot glasses is "200", then the math is deceptively easy.

ABV \* 20

The *2 PHP_ROUND_UP /2 portion is just a trick to force one decimal place.

**Comment by Skemp45** [comment]
http://zookeeperbrewing.com/

take a look

**Comment by Skemp45** [comment]
![tapside](https://f.cloud.github.com/assets/6318869/1879527/8fabead0-794b-11e3-8a72-b785f5d2fae8.png)

**Comment by Skemp45** [comment]
fixed in #24
